react-ui/src/project/index.js

```diff
@@ -199,7 +199,8 @@ const Project = () => {
                         </div>
                     ) : null}
                     {selectedValue === "Burndown Chart" && isBurndown ? (
-                        <SprintDetail sprintDetails={data.sprints} attributes={data.custom_attributes} token={auth} projectName={data.name} />
+                        // <SprintDetail sprintDetails={data.sprints} attributes={data.custom_attributes} token={auth} projectName={data.name} />
```

Comment:
Why it this commented out?

Reply:
It causes errors, I though it can be handled during integration

Reply:
Yeah we can handle during integration. It needs to be present in div element. Also we can place it in the Sprint Detail along with the other charts.